apps/cnquery/cmd/status.go

```diff
@@ -149,37 +149,42 @@ type ClientStatus struct {
 func (s Status) RenderCliStatus() {
 	if s.Client.Platform != nil {
 		agent := s.Client
-		log.Info().Msg("Platform:\t" + agent.Platform.Name)
-		log.Info().Msg("Version:\t" + agent.Platform.Version)
-		log.Info().Msg("Hostname:\t" + agent.Hostname)
-		log.Info().Msg("IP:\t\t" + agent.IP)
+		log.Info().Msg("Platform:\t\t" + agent.Platform.Name)
+		log.Info().Msg("Version:\t\t" + agent.Platform.Version)
+		log.Info().Msg("Hostname:\t\t" + agent.Hostname)
+		log.Info().Msg("IP:\t\t\t" + agent.IP)
 	} else {
 		log.Warn().Msg("could not determine client platform information")
 	}
 
-	log.Info().Msg("Time:\t\t" + s.Client.Timestamp)
-	log.Info().Msg("Version:\t" + cnquery.GetVersion() + " (API Version: " + cnquery.APIVersion() + ")")
+	log.Info().Msg("Time:\t\t\t" + s.Client.Timestamp)
+	log.Info().Msg("Version:\t\t" + cnquery.GetVersion() + " (API Version: " + cnquery.APIVersion() + ")")
+	log.Info().Msg("Latest Version:\t" + cnquery.GetLatestVersion())
+
+	if cnquery.GetVersion() != cnquery.GetLatestVersion() {
+		log.Warn().Msg("A newer version is available")
+	}
 
 	log.Info().Msg("API ConnectionConfig:\t" + s.Upstream.API.Endpoint)
-	log.Info().Msg("API Status:\t" + s.Upstream.API.Status)
-	log.Info().Msg("API Time:\t" + s.Upstream.API.Timestamp)
-	log.Info().Msg("API Version:\t" + s.Upstream.API.Version)
+	log.Info().Msg("API Status:\t\t" + s.Upstream.API.Status)
+	log.Info().Msg("API Time:\t\t" + s.Upstream.API.Timestamp)
+	log.Info().Msg("API Version:\t\t" + s.Upstream.API.Version)
 
 	if s.Upstream.API.Version != cnquery.APIVersion() {
 		log.Warn().Msg("API versions do not match, please update the client")
 	}
 
 	if len(s.Upstream.Features) > 0 {
-		log.Info().Msg("Features:\t" + strings.Join(s.Upstream.Features, ","))
+		log.Info().Msg("Features:\t\t" + strings.Join(s.Upstream.Features, ","))
 	}
 
 	if s.Client.ParentMrn == "" {
```

Comment:
shouldn't this check be `!=`?

Reply:
Hm, as long as we the numbers can't be negative it behaves the same I think

Reply:
I meant the `s.Client.ParentMrn == ""` block, it should probably be `s.Client.ParentMrn != ""`,

there is no point in showing an empty parent mrn

Reply:
Oh, you are right, changed it